.eslintrc.js

```diff
@@ -1,5 +1,7 @@
-const path = require('path');
-const Neutrino = require('neutrino');
-const api = new Neutrino([path.resolve('./neutrino-custom/development.js')]);
+const { Neutrino } = require('neutrino');
+const { neutrino } = require('./package.json');
 
-module.exports = api.custom.eslintrc();
+const api = Neutrino(neutrino.options);
+
+neutrino.use.map(middleware => api.use(require(middleware)));
```

Comment:
This differs slightly from the `.eslintrc.js` example given in the Neutrino docs. If this is preferred over the form given there, could you update the docs?
https://neutrino.js.org/middleware/neutrino-middleware-eslint/#eslintrc-config